src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -706,6 +706,11 @@ public with sharing class CRLP_RollupUI_SVC {
             // CMT data will generate a unique RecordName to use for the new record.
             CRLP_RollupCMT.Rollup rollupCMTObj = (CRLP_RollupCMT.Rollup) JSON.deserialize(rollupCMT, CRLP_RollupCMT.Rollup.class);
 
+            Boolean isRollupStateEnabled = new CRLP_ApiService().isRollupStateEnabled();
+            Boolean isRollupStale = false;
```

Comment:
`Stale` is a concept that is sort of unique to the DSO-NPSP Bridge. From the perspective of NPSP, maybe this could be described as `rollupChanged` which triggers an operation of something like `rollupChangedEvent` that eventually ends up in the api which knows what to do with the `rollupChanged` or `filterRuleChanged` event.